src/OrbitCaptureClient/GpuQueueSubmissionProcessor.cpp

```diff
@@ -23,31 +24,31 @@ std::vector<TimerInfo> GpuQueueSubmissionProcessor::ProcessGpuQueueSubmission(
       gpu_queue_submission.meta_info().pre_submission_cpu_timestamp();
   uint64_t post_submission_cpu_timestamp =
       gpu_queue_submission.meta_info().post_submission_cpu_timestamp();
-  const GpuJob* matching_gpu_job =
+  std::optional<GpuJob> matching_gpu_job_option =
```

Comment:
I find the name "option" a bit weird, shouldn't it be "optional"?

Reply:
Obsolete now.